exporters/trace/build.gradle

```diff
@@ -13,6 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+plugins {
```

Comment:
It is not needed, but this just generates a fat jar for exporter-trace dependency when we run `./gradlew shadowJar` task for the project so that exporter-trace can be consumed as an independent dependency. 

It is just for convenience.

Reply:
Please remove this change because it's unrelated to this PR.

Reply:
Should this have been removed?